### PR TITLE
feat(front): create input csv file

### PR DESCRIPTION
**Pull request by aymerickmichelet**
Enjoy!

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+        <link href="style.css" rel="stylesheet">
+        <title>Data quality</title>
+    </head>
+    <body>
+        <h1>Csv data quality</h1>
+        <h3>Give your csv, we say you if you have wrong data.</h3>
+        <div class="outer_quare">
+            <label id="inner_square" class="pointer inner_square" for="upload">
+                <div id="loader" class="hide lds-ring"><div></div><div></div><div></div><div></div></div>
+                <input id="upload" class="upload" type="file" onchange=switchStep() />
```
Comment (BullShitProd) [inline templates/index.html:14]: accept=".csv"

**Comment by aymerickmichelet** [comment]
I can add an icon file ^^

File: templates/script.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const Step = ["BEFORE", "LOADING", "RESULT"]
```
Comment (BullShitProd) [inline templates/script.js:1]: You don't use this const

File: templates/script.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+let index = 0;
+
+function switchStep(){
+    index < 2 ? index++ : index=0
```
Comment (BullShitProd) [inline templates/script.js:5]: index = 0

File: templates/script.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    displayStep()
+}
+
+
+function displayStep(){
+    let inner_square = document.getElementById("inner_square")
```
Comment (BullShitProd) [inline templates/script.js:11]: Use const (same for other let)

File: templates/script.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    let loader = document.getElementById("loader")
+    let upload = document.getElementById("upload")
+    let csv_icon = document.getElementById("csv_icon")
+    let label = document.getElementById("label")
+
+    if (index === 0){
```
Comment (BullShitProd) [inline templates/script.js:17]: if(!index) but this equality is ok

File: templates/script.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+const Step = ["BEFORE", "LOADING", "RESULT"]
+let index = 0;
+
+function switchStep(){
```
Comment (BullShitProd) [inline templates/script.js:4]: use arrow function :
` switchStep = () => {}`
It's modern JS 😀

File: templates/style.css [hunk 7]
```diff
@@ -0,0 +1,103 @@
+* {
+    font-family: "Source Sans Pro", Helvetica, Arial, sans-serif;
+    padding-top: 12px;
+}
+
+/* ------------------------------------------------- */
```
Comment (BullShitProd) [inline templates/style.css:6]: This comment isn't neccessary

Comment (BullShitProd) [reply templates/style.css:6]: at the limit you can put this : 
`/* ANIMATION */`

File: templates/style.css [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+    0% {
+        transform: rotate(0deg);
+    }
+    100% {
+        transform: rotate(360deg);
+    }
+}
+
+
+/* ------------------------------------------------- */
```
Comment (BullShitProd) [inline templates/style.css:45]: Same

File: templates/style.css [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+h1 {
```
Comment (BullShitProd) [inline templates/style.css:47]: Is better if you use a class but It's ok for this project

File: templates/style.css [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+    margin: 0;
+    padding: 0;
+    text-align: center;
+}
+
+h3 {
```
Comment (BullShitProd) [inline templates/style.css:53]: Same

File: templates/script.js (hunk 2 above)
Comment (aymerickmichelet) [reply templates/script.js:1]: Yeah I will remove the step and I do steps in comment

File: templates/script.js (hunk 3 above)
Comment (aymerickmichelet) [reply templates/script.js:5]: why not

File: templates/script.js (hunk 6 above)
Comment (aymerickmichelet) [reply templates/script.js:4]: Okay I will try this ^^

**Pull request closed without merging**